wallet-new/integration/WalletSpecs.hs

```diff
@@ -60,10 +61,10 @@ walletSpecs _ wc = do
 
             eresp <- getUtxoStatistics wc (walId wallet)
             utxoStatistics <- fmap wrData eresp `shouldPrism` _Right
-            let possibleBuckets = fmap show $ ( map (\toPower -> 10^toPower :: Word64) [(1::Word64)..16] ) ++ [45 * (10^(15::Word64))]
-            let histogram = map (\ x -> curry HistogramBarCount x 0) possibleBuckets
+            let possibleBuckets = fmap show $ (generateBounds Log10)
```

Comment:
parens or `$` are unnecessary here; could be either `fmap show (generateBounds Log10)` or `fmap show $ generateBounds Log10` or even `show <$> generateBounds Log10`

Reply:
fixed